cli/src/lib/libDefs.js

```diff
@@ -64,11 +64,23 @@ async function rebaseCacheRepo(verbose?: VerboseOutput) {
 }
 
 /**
+ * Utility wrapper for ensureCacheRepo with an update expiry of 0 hours.
+ */
+async function updateCacheRepo(verbose?: VerboseOutput): Promise<boolean> {
+  try {
```

Comment:
Actually, I am not 100% happy with the granularity of the different functions... initially I was thinking about refactoring stuff around so it would be more modular, but then I saw the jest tests and whatsoever... :-)

It would also be possible to just arrow-function the updateCacheRepo and try / catch the error in the command itself (instead of returning a boolean)